&lt;records xmlns="http://scientific.thomsonreuters.com/schema/wok5.4/public/FullRecord"&gt;
          &lt;REC r_id_disclaimer="ResearcherID data provided by Clarivate Analytics"&gt;&lt;UID&gt;MEDLINE:24452614&lt;/UID&gt;&lt;static_data&gt;&lt;summary&gt;&lt;EWUID&gt;&lt;WUID coll_id="MEDLINE"&gt;&lt;/WUID&gt;&lt;edition value="MEDLINE.MEDLINE"&gt;&lt;/edition&gt;&lt;/EWUID&gt;&lt;pub_info coverdate="2014-Jan-22" edate="2014-01-22" has_abstract="Y" medium="Print" model="Electronic" pubday="22" pubmonth="Jan" pubtype="Journal" pubyear="2014" sortdate="2014-01-22" vol="3"&gt;&lt;page begin="e93"&gt;e93&lt;/page&gt;&lt;/pub_info&gt;&lt;titles count="4"&gt;&lt;title type="item"&gt;Identifying druggable targets by protein microenvironments matching: application to transcription factors.&lt;/title&gt;&lt;title type="source"&gt;CPT: pharmacometrics &amp;amp; systems pharmacology&lt;/title&gt;&lt;title type="abbrev_iso"&gt;CPT Pharmacometrics Syst Pharmacol&lt;/title&gt;&lt;title type="source_abbrev"&gt;CPT Pharmacometrics Syst Pharmacol&lt;/title&gt;&lt;/titles&gt;&lt;names count="2"&gt;&lt;name display="Y" role="author" seq_no="1"&gt;&lt;display_name&gt;Liu, T&lt;/display_name&gt;&lt;full_name&gt;Liu, T&lt;/full_name&gt;&lt;initials&gt;T&lt;/initials&gt;&lt;/name&gt;&lt;name display="Y" role="author" seq_no="2"&gt;&lt;display_name&gt;Altman, R B&lt;/display_name&gt;&lt;full_name&gt;Altman, R B&lt;/full_name&gt;&lt;initials&gt;RB&lt;/initials&gt;&lt;/name&gt;&lt;/names&gt;&lt;doctypes count="1"&gt;&lt;doctype&gt;Journal Article&lt;/doctype&gt;&lt;/doctypes&gt;&lt;/summary&gt;&lt;fullrecord_metadata&gt;&lt;languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/languages&gt;&lt;normalized_languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/normalized_languages&gt;&lt;normalized_doctypes count="1"&gt;&lt;doctype&gt;Article&lt;/doctype&gt;&lt;/normalized_doctypes&gt;&lt;addresses count="1"&gt;&lt;address_name&gt;&lt;address_spec addr_no="1"&gt;&lt;full_address&gt;Department of Genetics, Stanford University, Stanford, California, USA.&lt;/full_address&gt;&lt;/address_spec&gt;&lt;/address_name&gt;&lt;/addresses&gt;&lt;fund_ack&gt;&lt;grants complete="Y" count="3"&gt;&lt;grant&gt;&lt;grant_agency&gt;NIGMS NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;R01 GM102365&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;GM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NLM NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;R01 LM005652&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;LM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NIGMS NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;U54 GM072970&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;GM&lt;/acronym&gt;&lt;/grant&gt;&lt;/grants&gt;&lt;/fund_ack&gt;&lt;abstracts count="1"&gt;&lt;abstract&gt;&lt;abstract_text&gt;&lt;p&gt;Druggability of a protein is its potential to be modulated by drug-like molecules. It is important in the target selection phase. We hypothesize that: (i) known drug-binding sites contain advantageous physicochemical properties for drug binding, or &amp;quot;druggable microenvironments&amp;quot; and (ii) given a target, the presence of multiple druggable microenvironments similar to those seen previously is associated with a high likelihood of druggability. We developed DrugFEATURE to quantify druggability by assessing the microenvironments in potential small-molecule binding sites. We benchmarked DrugFEATURE using two data sets. One data set measures druggability using NMR-based screening. DrugFEATURE correlates well with this metric. The second data set is based on historical drug discovery outcomes. Using the DrugFEATURE cutoffs derived from the first, we accurately discriminated druggable and difficult targets in the second. We further identified novel druggable transcription factors with implications for cancer therapy. DrugFEATURE provides useful insight for drug discovery, by evaluating druggability and suggesting specific regions for interacting with drug-like molecules.CPT: Pharmacometrics Systems Pharmacology (2014) 3, e93; doi:10.1038/psp.2013.66; published online 22 January 2014. &lt;/p&gt;&lt;/abstract_text&gt;&lt;/abstract&gt;&lt;/abstracts&gt;&lt;/fullrecord_metadata&gt;&lt;item xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Owner="NLM" Status="PubMed-not-MEDLINE" coll_id="MEDLINE" xsi:type="itemType_medline"&gt;&lt;MedlineJournalInfo&gt;&lt;Country&gt;United States&lt;/Country&gt;&lt;NlmUniqueID&gt;101580011&lt;/NlmUniqueID&gt;&lt;ISSNLinking&gt;2163-8306&lt;/ISSNLinking&gt;&lt;/MedlineJournalInfo&gt;&lt;DateCreated&gt;2014-01-23&lt;/DateCreated&gt;&lt;DateCompleted&gt;2014-01-23&lt;/DateCompleted&gt;&lt;DateRevised&gt;2016-10-19&lt;/DateRevised&gt;&lt;Affiliation&gt;Department of Genetics, Stanford University, Stanford, California, USA.&lt;/Affiliation&gt;&lt;OtherID Source="NLM"&gt;PMC3910014&lt;/OtherID&gt;&lt;/item&gt;&lt;/static_data&gt;&lt;dynamic_data&gt;&lt;citation_related&gt;&lt;tc_list&gt;&lt;silo_tc coll_id="MEDLINE" local_count="0"&gt;&lt;/silo_tc&gt;&lt;/tc_list&gt;&lt;/citation_related&gt;&lt;cluster_related&gt;&lt;identifiers&gt;&lt;identifier type="doi" value="10.1038/psp.2013.66"&gt;&lt;/identifier&gt;&lt;identifier type="pmid" value="MEDLINE:24452614"&gt;&lt;/identifier&gt;&lt;/identifiers&gt;&lt;/cluster_related&gt;&lt;/dynamic_data&gt;&lt;/REC&gt;
          &lt;REC r_id_disclaimer="ResearcherID data provided by Clarivate Analytics"&gt;&lt;UID&gt;MEDLINE:24303232&lt;/UID&gt;&lt;static_data&gt;&lt;summary&gt;&lt;EWUID&gt;&lt;WUID coll_id="MEDLINE"&gt;&lt;/WUID&gt;&lt;edition value="MEDLINE.MEDLINE"&gt;&lt;/edition&gt;&lt;/EWUID&gt;&lt;pub_info coverdate="2013" edate="2013-03-18" has_abstract="Y" medium="Print" model="Electronic-eCollection" pubtype="Journal" pubyear="2013" sortdate="2013-01-01" vol="2013"&gt;&lt;page begin="34" end="8"&gt;34-8&lt;/page&gt;&lt;/pub_info&gt;&lt;titles count="4"&gt;&lt;title type="item"&gt;Mining for clinical expertise in (undocumented) order sets to power an order suggestion system.&lt;/title&gt;&lt;title type="source"&gt;AMIA Joint Summits on Translational Science proceedings. AMIA Joint Summits on Translational Science&lt;/title&gt;&lt;title type="abbrev_iso"&gt;AMIA Jt Summits Transl Sci Proc&lt;/title&gt;&lt;title type="source_abbrev"&gt;AMIA Jt Summits Transl Sci Proc&lt;/title&gt;&lt;/titles&gt;&lt;names count="2"&gt;&lt;name display="Y" role="author" seq_no="1"&gt;&lt;display_name&gt;Chen, Jonathan H&lt;/display_name&gt;&lt;full_name&gt;Chen, Jonathan H&lt;/full_name&gt;&lt;initials&gt;JH&lt;/initials&gt;&lt;/name&gt;&lt;name display="Y" role="author" seq_no="2"&gt;&lt;display_name&gt;Altman, Russ B&lt;/display_name&gt;&lt;full_name&gt;Altman, Russ B&lt;/full_name&gt;&lt;initials&gt;RB&lt;/initials&gt;&lt;/name&gt;&lt;/names&gt;&lt;doctypes count="1"&gt;&lt;doctype&gt;Journal Article&lt;/doctype&gt;&lt;/doctypes&gt;&lt;/summary&gt;&lt;fullrecord_metadata&gt;&lt;languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/languages&gt;&lt;normalized_languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/normalized_languages&gt;&lt;normalized_doctypes count="1"&gt;&lt;doctype&gt;Article&lt;/doctype&gt;&lt;/normalized_doctypes&gt;&lt;addresses count="1"&gt;&lt;address_name&gt;&lt;address_spec addr_no="1"&gt;&lt;full_address&gt;Department of Medicine, Stanford University, Stanford, CA 94305, USA.&lt;/full_address&gt;&lt;/address_spec&gt;&lt;/address_name&gt;&lt;/addresses&gt;&lt;fund_ack&gt;&lt;grants complete="Y" count="5"&gt;&lt;grant&gt;&lt;grant_agency&gt;NIGMS NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;R24 GM061374&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;GM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NLM NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;R01 LM005652&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;LM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NCRR NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;UL1 RR025744&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;RR&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NIGMS NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;R01 GM102365&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;GM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NCATS NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;UL1 TR001085&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;TR&lt;/acronym&gt;&lt;/grant&gt;&lt;/grants&gt;&lt;/fund_ack&gt;&lt;abstracts count="1"&gt;&lt;abstract&gt;&lt;abstract_text&gt;&lt;p&gt;Physician orders, the concrete manifestation of clinical decision making, are enhanced by the distribution of clinical expertise in the form of order sets and corollary orders. Conventional order sets are top-down distributed by committees of experts, limited by the cost of manual development, maintenance, and limited end-user awareness. An alternative explored here applies statistical data-mining to physician order data (&amp;gt;330K order instances from &amp;gt;1.4K inpatient encounters) to extract clinical expertise from the bottom-up. This powers a corollary order suggestion engine using techniques analogous to commercial product recommendation systems (e.g., Amazon.com&amp;apos;s &amp;quot;Customers who bought this&amp;quot; feature). Compared to a simple benchmark, the item-based association method illustrated here improves order prediction precision from 13% to 18% and further to 28% by incorporating information on the temporal relationship between orders. Incorporating statistics on conditional order frequency ratios further refines recommendations beyond just &amp;quot;common&amp;quot; orders to those relevant to a specific clinical context. &lt;/p&gt;&lt;/abstract_text&gt;&lt;/abstract&gt;&lt;/abstracts&gt;&lt;/fullrecord_metadata&gt;&lt;item xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Owner="NLM" Status="PubMed-not-MEDLINE" coll_id="MEDLINE" xsi:type="itemType_medline"&gt;&lt;MedlineJournalInfo&gt;&lt;Country&gt;United States&lt;/Country&gt;&lt;NlmUniqueID&gt;101539486&lt;/NlmUniqueID&gt;&lt;/MedlineJournalInfo&gt;&lt;DateCreated&gt;2013-12-04&lt;/DateCreated&gt;&lt;DateCompleted&gt;2013-12-04&lt;/DateCompleted&gt;&lt;DateRevised&gt;2017-02-22&lt;/DateRevised&gt;&lt;Affiliation&gt;Department of Medicine, Stanford University, Stanford, CA 94305, USA.&lt;/Affiliation&gt;&lt;CommentsCorrectionsList&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;AMIA Annu Symp Proc. 2009 Nov 14;2009:391-5&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20351886&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;J Am Med Inform Assoc. 2003 Nov-Dec;10(6):523-30&lt;/RefSource&gt;&lt;PMID Version="1"&gt;12925543&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;AMIA Annu Symp Proc. 2006;:819-23&lt;/RefSource&gt;&lt;PMID Version="1"&gt;17238455&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;J Am Med Inform Assoc. 1997 Sep-Oct;4(5):364-75&lt;/RefSource&gt;&lt;PMID Version="1"&gt;9292842&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;AMIA Annu Symp Proc. 2009 Nov 14;2009:244-8&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20351858&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;AMIA Annu Symp Proc. 2009 Nov 14;2009:333-7&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20351875&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Med Inform Internet Med. 2001 Jan-Mar;26(1):25-33&lt;/RefSource&gt;&lt;PMID Version="1"&gt;11583406&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Arch Intern Med. 2003 Jun 23;163(12):1409-16&lt;/RefSource&gt;&lt;PMID Version="1"&gt;12824090&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;/CommentsCorrectionsList&gt;&lt;OtherID Source="NLM"&gt;PMC3845792&lt;/OtherID&gt;&lt;/item&gt;&lt;/static_data&gt;&lt;dynamic_data&gt;&lt;citation_related&gt;&lt;tc_list&gt;&lt;silo_tc coll_id="MEDLINE" local_count="0"&gt;&lt;/silo_tc&gt;&lt;/tc_list&gt;&lt;/citation_related&gt;&lt;cluster_related&gt;&lt;identifiers&gt;&lt;identifier type="pmid" value="MEDLINE:24303232"&gt;&lt;/identifier&gt;&lt;/identifiers&gt;&lt;/cluster_related&gt;&lt;/dynamic_data&gt;&lt;/REC&gt;
          &lt;REC r_id_disclaimer="ResearcherID data provided by Clarivate Analytics"&gt;&lt;UID&gt;MEDLINE:21121048&lt;/UID&gt;&lt;static_data&gt;&lt;summary&gt;&lt;EWUID&gt;&lt;WUID coll_id="MEDLINE"&gt;&lt;/WUID&gt;&lt;edition value="MEDLINE.MEDLINE"&gt;&lt;/edition&gt;&lt;/EWUID&gt;&lt;pub_info coverdate="2011" has_abstract="Y" medium="Internet" model="Print" pubtype="Journal" pubyear="2011" sortdate="2011-01-01"&gt;&lt;page begin="205" end="11"&gt;205-11&lt;/page&gt;&lt;/pub_info&gt;&lt;titles count="4"&gt;&lt;title type="item"&gt;Structural insights into pre-translocation ribosome motions.&lt;/title&gt;&lt;title type="source"&gt;Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing&lt;/title&gt;&lt;title type="abbrev_iso"&gt;Pac Symp Biocomput&lt;/title&gt;&lt;title type="source_abbrev"&gt;Pac Symp Biocomput&lt;/title&gt;&lt;/titles&gt;&lt;names count="2"&gt;&lt;name display="Y" role="author" seq_no="1"&gt;&lt;display_name&gt;Flores, Samuel Coulbourn&lt;/display_name&gt;&lt;full_name&gt;Flores, Samuel Coulbourn&lt;/full_name&gt;&lt;initials&gt;SC&lt;/initials&gt;&lt;/name&gt;&lt;name display="Y" role="author" seq_no="2"&gt;&lt;display_name&gt;Altman, Russ&lt;/display_name&gt;&lt;full_name&gt;Altman, Russ&lt;/full_name&gt;&lt;initials&gt;R&lt;/initials&gt;&lt;/name&gt;&lt;/names&gt;&lt;doctypes count="1"&gt;&lt;doctype&gt;Journal Article&lt;/doctype&gt;&lt;/doctypes&gt;&lt;/summary&gt;&lt;fullrecord_metadata&gt;&lt;languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/languages&gt;&lt;normalized_languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/normalized_languages&gt;&lt;normalized_doctypes count="1"&gt;&lt;doctype&gt;Article&lt;/doctype&gt;&lt;/normalized_doctypes&gt;&lt;addresses count="1"&gt;&lt;address_name&gt;&lt;address_spec addr_no="1"&gt;&lt;full_address&gt;Bioengineering Department, Stanford University, James H Clark Center S172 MC:5448, Stanford, California 94305, USA. samuelflorec@gmail.com&lt;/full_address&gt;&lt;/address_spec&gt;&lt;/address_name&gt;&lt;/addresses&gt;&lt;category_info&gt;&lt;headings count="1"&gt;&lt;heading&gt;Science &amp;amp; Technology&lt;/heading&gt;&lt;/headings&gt;&lt;subheadings count="2"&gt;&lt;subheading&gt;Life Sciences &amp;amp; Biomedicine&lt;/subheading&gt;&lt;subheading&gt;Technology&lt;/subheading&gt;&lt;/subheadings&gt;&lt;subjects count="16"&gt;&lt;subject ascatype="traditional" code="QU"&gt;MICROBIOLOGY&lt;/subject&gt;&lt;subject ascatype="traditional" code="CU"&gt;BIOLOGY&lt;/subject&gt;&lt;subject ascatype="traditional" code="RA"&gt;MICROSCOPY&lt;/subject&gt;&lt;subject ascatype="traditional"&gt;IMAGING SCIENCE PHOTOGRAPHIC TECHNOLOGY&lt;/subject&gt;&lt;subject ascatype="traditional"&gt;BIOCHEMISTRY MOLECULAR BIOLOGY&lt;/subject&gt;&lt;subject ascatype="traditional" code="UM"&gt;PHYSIOLOGY&lt;/subject&gt;&lt;subject ascatype="traditional"&gt;GENETICS HEREDITY&lt;/subject&gt;&lt;subject ascatype="traditional" code="DR"&gt;CELL BIOLOGY&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Microbiology&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Life Sciences &amp;amp; Biomedicine - Other Topics&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Microscopy&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Imaging Science &amp;amp; Photographic Technology&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Biochemistry &amp;amp; Molecular Biology&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Physiology&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Genetics &amp;amp; Heredity&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Cell Biology&lt;/subject&gt;&lt;/subjects&gt;&lt;/category_info&gt;&lt;fund_ack&gt;&lt;grants complete="Y" count="2"&gt;&lt;grant&gt;&lt;grant_agency&gt;NLM NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;R01 LM005652&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;LM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NIGMS NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;U54 GM072970&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;GM&lt;/acronym&gt;&lt;/grant&gt;&lt;/grants&gt;&lt;/fund_ack&gt;&lt;abstracts count="1"&gt;&lt;abstract&gt;&lt;abstract_text&gt;&lt;p&gt;Subsequent to the peptidyl transfer step of the translation elongation cycle, the initially formed pre-translocation ribosome, which we refer to here as R(1), undergoes a ratchet-like intersubunit rotation in order to sample a rotated conformation, referred to here as R(F), that is an obligatory intermediate in the translocation of tRNAs and mRNA through the ribosome during the translocation step of the translation elongation cycle. R(F) and the R(1) to R(F) transition are currently the subject of intense research, driven in part by the potential for developing novel antibiotics which trap R(F) or confound the R(1) to R(F) transition. Currently lacking a 3D atomic structure of the R(F) endpoint of the transition, as well as a preliminary conformational trajectory connecting R(1) and R(F), the dynamics of the mechanistically crucial R(1) to R(F) transition remain elusive. The current literature reports fitting of only a few ribosomal RNA (rRNA) and ribosomal protein (r-protein) components into cryogenic electron microscopy (cryo-EM) reconstructions of the Escherichia coli ribosome in RF. In this work we now fit the entire Thermus thermophilus 16S and 23S rRNAs and most of the remaining T. thermophilus r-proteins into a cryo-EM reconstruction of the E. coli ribosome in R(F) in order to build an almost complete model of the T. thermophilus ribosome in R(F) thus allowing a more detailed view of this crucial conformation. The resulting model validates key predictions from the published literature; in particular it recovers intersubunit bridges known to be maintained throughout the R(1) to R(F) transition and results in new intersubunit bridges that are predicted to exist only in R(F). In addition, we use a recently reported E. coli ribosome structure, apparently trapped in an intermediate state along the R(1) to R(F) transition pathway, referred to here as R(2), as a guide to generate a T. thermophilus ribosome in the R(2) state. This demonstrates a multiresolution method for morphing large complexes and provides us with a structural model of R(2) in the species of interest. The generated structural models form the basis for probing the motion of the deacylated tRNA bound at the peptidyl-tRNA binding site (P site) of the pre-translocation ribosome as it moves from its so-called classical P/P configuration to its so-called hybrid P/E configuration as part of the R(1) to R(F) transition. We create a dynamic model of this process which provides structural insights into the functional significance of R(2) as well as detailed atomic information to guide the design of further experiments. The results suggest extensibility to other steps of protein synthesis as well as to spatially larger systems.&lt;/p&gt;&lt;/abstract_text&gt;&lt;/abstract&gt;&lt;/abstracts&gt;&lt;/fullrecord_metadata&gt;&lt;item xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Owner="NLM" Status="MEDLINE" coll_id="MEDLINE" xsi:type="itemType_medline"&gt;&lt;MedlineJournalInfo&gt;&lt;Country&gt;United States&lt;/Country&gt;&lt;NlmUniqueID&gt;9711271&lt;/NlmUniqueID&gt;&lt;/MedlineJournalInfo&gt;&lt;DateCreated&gt;2010-12-01&lt;/DateCreated&gt;&lt;DateCompleted&gt;2013-11-13&lt;/DateCompleted&gt;&lt;DateRevised&gt;2016-10-25&lt;/DateRevised&gt;&lt;Affiliation&gt;Bioengineering Department, Stanford University, James H Clark Center S172 MC:5448, Stanford, California 94305, USA. samuelflorec@gmail.com&lt;/Affiliation&gt;&lt;ChemicalList&gt;&lt;Chemical&gt;&lt;RegistryNumber&gt;0&lt;/RegistryNumber&gt;&lt;NameOfSubstance UI="D001426"&gt;Bacterial Proteins&lt;/NameOfSubstance&gt;&lt;/Chemical&gt;&lt;Chemical&gt;&lt;RegistryNumber&gt;0&lt;/RegistryNumber&gt;&lt;NameOfSubstance UI="D012329"&gt;RNA, Bacterial&lt;/NameOfSubstance&gt;&lt;/Chemical&gt;&lt;Chemical&gt;&lt;RegistryNumber&gt;0&lt;/RegistryNumber&gt;&lt;NameOfSubstance UI="D012335"&gt;RNA, Ribosomal&lt;/NameOfSubstance&gt;&lt;/Chemical&gt;&lt;Chemical&gt;&lt;RegistryNumber&gt;0&lt;/RegistryNumber&gt;&lt;NameOfSubstance UI="D012269"&gt;Ribosomal Proteins&lt;/NameOfSubstance&gt;&lt;/Chemical&gt;&lt;/ChemicalList&gt;&lt;CitationSubset&gt;IM&lt;/CitationSubset&gt;&lt;CommentsCorrectionsList&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;J Mol Biol. 2002 Mar 29;317(3):375-84&lt;/RefSource&gt;&lt;PMID Version="1"&gt;11922671&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;J Comput Chem. 2004 Oct;25(13):1605-12&lt;/RefSource&gt;&lt;PMID Version="1"&gt;15264254&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Structure. 2005 Mar;13(3):389-400&lt;/RefSource&gt;&lt;PMID Version="1"&gt;15766540&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Science. 2005 Nov 4;310(5749):827-34&lt;/RefSource&gt;&lt;PMID Version="1"&gt;16272117&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Science. 2006 Sep 29;313(5795):1935-42&lt;/RefSource&gt;&lt;PMID Version="1"&gt;16959973&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Mol Cell. 2007 Mar 9;25(5):751-64&lt;/RefSource&gt;&lt;PMID Version="1"&gt;17349960&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Curr Opin Struct Biol. 2007 Apr;17(2):192-8&lt;/RefSource&gt;&lt;PMID Version="1"&gt;17383173&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Curr Opin Struct Biol. 2007 Jun;17(3):302-9&lt;/RefSource&gt;&lt;PMID Version="1"&gt;17574829&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;J Mol Biol. 2007 Nov 2;373(4):1058-70&lt;/RefSource&gt;&lt;PMID Version="1"&gt;17897673&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Structure. 2008 May;16(5):673-83&lt;/RefSource&gt;&lt;PMID Version="1"&gt;18462672&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nucleic Acids Res. 2009 Jul;37(12):4033-42&lt;/RefSource&gt;&lt;PMID Version="1"&gt;19417061&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Science. 2009 Aug 21;325(5943):1014-7&lt;/RefSource&gt;&lt;PMID Version="1"&gt;19696352&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Pac Symp Biocomput. 2010;:216-27&lt;/RefSource&gt;&lt;PMID Version="1"&gt;19908374&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;RNA. 2010 Sep;16(9):1769-78&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20651028&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;IEEE/ACM Trans Comput Biol Bioinform. 2011 Sep-Oct;8(5):1247-57&lt;/RefSource&gt;&lt;PMID Version="1"&gt;21778523&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Science. 2001 May 4;292(5518):883-96&lt;/RefSource&gt;&lt;PMID Version="1"&gt;11283358&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;J Struct Biol. 2001 Feb-Mar;133(2-3):193-202&lt;/RefSource&gt;&lt;PMID Version="1"&gt;11472090&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;/CommentsCorrectionsList&gt;&lt;MeshHeadingList&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D001426"&gt;Bacterial Proteins&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000737"&gt;chemistry&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000378"&gt;metabolism&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;D12.776.097&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y03&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y08.040&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D019295"&gt;Computational Biology&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;H01.158.273.180&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.313.124&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D020285"&gt;Cryoelectron Microscopy&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;E01.370.350.515.402.150&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.595.402.150&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D004926"&gt;Escherichia coli&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000378"&gt;metabolism&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000648"&gt;ultrastructure&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;B03.440.450.425.325.300&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;B03.660.250.150.180.100&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y08.040&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y02.020.020&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D007091"&gt;Image Processing, Computer-Assisted&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.224.308&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D008958"&gt;Models, Molecular&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.599.595&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D008968"&gt;Molecular Conformation&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;G02.111.570.820&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D009068"&gt;Movement&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;G07.700.560&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;G11.427.590.530&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D012329"&gt;RNA, Bacterial&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000737"&gt;chemistry&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000378"&gt;metabolism&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;D13.444.735.473&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y03&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y08.040&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D012335"&gt;RNA, Ribosomal&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000737"&gt;chemistry&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000378"&gt;metabolism&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;D13.444.735.686&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y03&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y08.040&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D012269"&gt;Ribosomal Proteins&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000737"&gt;chemistry&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000378"&gt;metabolism&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;D12.776.835&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y03&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y08.040&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D012270"&gt;Ribosomes&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="Y" UI="Q000737"&gt;chemistry&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="Y" UI="Q000378"&gt;metabolism&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000648"&gt;ultrastructure&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;A11.284.430.214.190.875.811&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="Y"&gt;Y03&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="Y"&gt;Y08.040&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y02.020.020&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D016963"&gt;Thermus thermophilus&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000378"&gt;metabolism&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000648"&gt;ultrastructure&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;B03.440.400.425.875.875&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y08.040&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y02.020.020&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;/MeshHeadingList&gt;&lt;OtherID Source="NLM"&gt;NIHMS377523&lt;/OtherID&gt;&lt;OtherID Source="NLM"&gt;PMC3371265&lt;/OtherID&gt;&lt;/item&gt;&lt;/static_data&gt;&lt;dynamic_data&gt;&lt;citation_related&gt;&lt;tc_list&gt;&lt;silo_tc coll_id="MEDLINE" local_count="0"&gt;&lt;/silo_tc&gt;&lt;/tc_list&gt;&lt;/citation_related&gt;&lt;cluster_related&gt;&lt;identifiers&gt;&lt;identifier type="issn" value="2335-6936"&gt;&lt;/identifier&gt;&lt;identifier type="pmid" value="MEDLINE:21121048"&gt;&lt;/identifier&gt;&lt;/identifiers&gt;&lt;/cluster_related&gt;&lt;/dynamic_data&gt;&lt;/REC&gt;
          &lt;REC r_id_disclaimer="ResearcherID data provided by Clarivate Analytics"&gt;&lt;UID&gt;MEDLINE:26776202&lt;/UID&gt;&lt;static_data&gt;&lt;summary&gt;&lt;EWUID&gt;&lt;WUID coll_id="MEDLINE"&gt;&lt;/WUID&gt;&lt;edition value="MEDLINE.MEDLINE"&gt;&lt;/edition&gt;&lt;/EWUID&gt;&lt;pub_info coverdate="2016" has_abstract="Y" medium="Internet" model="Print" pubtype="Journal" pubyear="2016" sortdate="2016-01-01" vol="21"&gt;&lt;page begin="381" end="92"&gt;381-92&lt;/page&gt;&lt;/pub_info&gt;&lt;titles count="4"&gt;&lt;title type="item"&gt;SEPARATING THE CAUSES AND CONSEQUENCES IN DISEASE TRANSCRIPTOME.&lt;/title&gt;&lt;title type="source"&gt;Pacific Symposium on Biocomputing. Pacific Symposium on Biocomputing&lt;/title&gt;&lt;title type="abbrev_iso"&gt;Pac Symp Biocomput&lt;/title&gt;&lt;title type="source_abbrev"&gt;Pac Symp Biocomput&lt;/title&gt;&lt;/titles&gt;&lt;names count="3"&gt;&lt;name display="Y" role="author" seq_no="1"&gt;&lt;display_name&gt;Li, Yong Fuga&lt;/display_name&gt;&lt;full_name&gt;Li, Yong Fuga&lt;/full_name&gt;&lt;initials&gt;YF&lt;/initials&gt;&lt;/name&gt;&lt;name display="Y" role="author" seq_no="2"&gt;&lt;display_name&gt;Xin, Fuxiao&lt;/display_name&gt;&lt;full_name&gt;Xin, Fuxiao&lt;/full_name&gt;&lt;initials&gt;F&lt;/initials&gt;&lt;/name&gt;&lt;name display="Y" role="author" seq_no="3"&gt;&lt;display_name&gt;Altman, Russ B&lt;/display_name&gt;&lt;full_name&gt;Altman, Russ B&lt;/full_name&gt;&lt;initials&gt;RB&lt;/initials&gt;&lt;/name&gt;&lt;/names&gt;&lt;doctypes count="3"&gt;&lt;doctype&gt;Journal Article&lt;/doctype&gt;&lt;doctype&gt;Research Support, N.I.H., Extramural&lt;/doctype&gt;&lt;doctype&gt;Research Support, Non-U.S. Gov&amp;apos;t&lt;/doctype&gt;&lt;/doctypes&gt;&lt;/summary&gt;&lt;fullrecord_metadata&gt;&lt;languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/languages&gt;&lt;normalized_languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/normalized_languages&gt;&lt;normalized_doctypes count="2"&gt;&lt;doctype&gt;Article&lt;/doctype&gt;&lt;doctype&gt;Other&lt;/doctype&gt;&lt;/normalized_doctypes&gt;&lt;addresses count="1"&gt;&lt;address_name&gt;&lt;address_spec addr_no="1"&gt;&lt;full_address&gt;Department of Bioengineering, Stanford University, USA2Stanford Genome Technology Center, Stanford University, USA.&lt;/full_address&gt;&lt;/address_spec&gt;&lt;/address_name&gt;&lt;/addresses&gt;&lt;category_info&gt;&lt;headings count="1"&gt;&lt;heading&gt;Science &amp;amp; Technology&lt;/heading&gt;&lt;/headings&gt;&lt;subheadings count="2"&gt;&lt;subheading&gt;Physical Sciences&lt;/subheading&gt;&lt;subheading&gt;Life Sciences &amp;amp; Biomedicine&lt;/subheading&gt;&lt;/subheadings&gt;&lt;subjects count="10"&gt;&lt;subject ascatype="traditional" code="PQ"&gt;MATHEMATICS&lt;/subject&gt;&lt;subject ascatype="traditional" code="CU"&gt;BIOLOGY&lt;/subject&gt;&lt;subject ascatype="traditional"&gt;ENDOCRINOLOGY METABOLISM&lt;/subject&gt;&lt;subject ascatype="traditional"&gt;GENETICS HEREDITY&lt;/subject&gt;&lt;subject ascatype="traditional" code="VE"&gt;PSYCHIATRY&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Mathematics&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Life Sciences &amp;amp; Biomedicine - Other Topics&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Endocrinology &amp;amp; Metabolism&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Genetics &amp;amp; Heredity&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Psychiatry&lt;/subject&gt;&lt;/subjects&gt;&lt;/category_info&gt;&lt;fund_ack&gt;&lt;grants complete="Y" count="8"&gt;&lt;grant&gt;&lt;grant_agency&gt;NHLBI NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;U54 HL117798&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;HL&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NLM NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;R01 LM005652&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;LM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NIMH NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;P50 MH094267&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;MH&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NIMH NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;MH094267&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;MH&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NIGMS NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;GM102365&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;GM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NIGMS NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;R01 GM102365&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;GM&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NHLBI NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;HL117798&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;HL&lt;/acronym&gt;&lt;/grant&gt;&lt;grant&gt;&lt;grant_agency&gt;NLM NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;LM005652&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;LM&lt;/acronym&gt;&lt;/grant&gt;&lt;/grants&gt;&lt;/fund_ack&gt;&lt;abstracts count="1"&gt;&lt;abstract&gt;&lt;abstract_text&gt;&lt;p&gt;The causes of complex diseases are multifactorial and the phenotypes of complex diseases are typically heterogeneous, posting significant challenges for both the experiment design and statistical inference in the study of such diseases. Transcriptome profiling can potentially provide key insights on the pathogenesis of diseases, but the signals from the disease causes and consequences are intertwined, leaving it to speculations what are likely causal. Genome-wide association study on the other hand provides direct evidences on the potential genetic causes of diseases, but it does not provide a comprehensive view of disease pathogenesis, and it has difficulties in detecting the weak signals from individual genes. Here we propose an approach diseaseExPatho that combines transcriptome data, regulome knowledge, and GWAS results if available, for separating the causes and consequences in the disease transcriptome. DiseaseExPatho computationally deconvolutes the expression data into gene expression modules, hierarchically ranks the modules based on regulome using a novel algorithm, and given GWAS data, it directly labels the potential causal gene modules based on their correlations with genome-wide gene-disease associations. Strikingly, we observed that the putative causal modules are not necessarily differentially expressed in disease, while the other modules can show strong differential expression without enrichment of top GWAS variations. On the other hand, we showed that the regulatory network based module ranking prioritized the putative causal modules consistently in 6 diseases. We suggest that the approach is applicable to other common and rare complex diseases to prioritize causal pathways with or without genome-wide association studies. &lt;/p&gt;&lt;/abstract_text&gt;&lt;/abstract&gt;&lt;/abstracts&gt;&lt;/fullrecord_metadata&gt;&lt;item xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Owner="NLM" Status="MEDLINE" coll_id="MEDLINE" xsi:type="itemType_medline"&gt;&lt;MedlineJournalInfo&gt;&lt;Country&gt;United States&lt;/Country&gt;&lt;NlmUniqueID&gt;9711271&lt;/NlmUniqueID&gt;&lt;/MedlineJournalInfo&gt;&lt;DateCreated&gt;2016-01-18&lt;/DateCreated&gt;&lt;DateCompleted&gt;2016-10-24&lt;/DateCompleted&gt;&lt;DateRevised&gt;2016-12-30&lt;/DateRevised&gt;&lt;Affiliation&gt;Department of Bioengineering, Stanford University, USA2Stanford Genome Technology Center, Stanford University, USA.&lt;/Affiliation&gt;&lt;CitationSubset&gt;IM&lt;/CitationSubset&gt;&lt;CommentsCorrectionsList&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nucleic Acids Res. 2005 Jan 1;33(Database issue):D428-32&lt;/RefSource&gt;&lt;PMID Version="1"&gt;15608231&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Rev Genet. 2004 Feb;5(2):89-100&lt;/RefSource&gt;&lt;PMID Version="1"&gt;14735120&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Annu Rev Genomics Hum Genet. 2005;6:381-406&lt;/RefSource&gt;&lt;PMID Version="1"&gt;16124867&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nucleic Acids Res. 2006 Jan 1;34(Database issue):D74-81&lt;/RefSource&gt;&lt;PMID Version="1"&gt;16381970&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Proc Natl Acad Sci U S A. 2006 Oct 3;103(40):14724-31&lt;/RefSource&gt;&lt;PMID Version="1"&gt;17003135&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Genet. 2007 Oct;39(10):1181-6&lt;/RefSource&gt;&lt;PMID Version="1"&gt;17898773&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Rev Genet. 2008 May;9(5):356-69&lt;/RefSource&gt;&lt;PMID Version="1"&gt;18398418&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Mol Syst Biol. 2008;4:189&lt;/RefSource&gt;&lt;PMID Version="1"&gt;18463613&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Biotechnol. 2008 Oct;26(10):1125-33&lt;/RefSource&gt;&lt;PMID Version="1"&gt;18846086&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Genomics. 2008 Nov;92(5):255-64&lt;/RefSource&gt;&lt;PMID Version="1"&gt;18703132&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nucleic Acids Res. 2009 Jan;37(Database issue):D674-9&lt;/RefSource&gt;&lt;PMID Version="1"&gt;18832364&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Rev Genet. 2009 Jan;10(1):57-63&lt;/RefSource&gt;&lt;PMID Version="1"&gt;19015660&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Methods. 2008 Sep;5(9):829-34&lt;/RefSource&gt;&lt;PMID Version="1"&gt;19160518&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Rev Genet. 2009 Apr;10(4):252-63&lt;/RefSource&gt;&lt;PMID Version="1"&gt;19274049&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;PLoS One. 2009;4(8):e6511&lt;/RefSource&gt;&lt;PMID Version="1"&gt;19652704&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Biostatistics. 2010 Apr;11(2):242-53&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20097884&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Am J Hum Genet. 2010 Apr 9;86(4):581-91&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20346437&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Proc Natl Acad Sci U S A. 2010 Apr 13;107(15):6841-6&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20351254&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Proc Natl Acad Sci U S A. 2010 May 18;107(20):9186-91&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20439753&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;J Biomed Inform. 2010 Dec;43(6):932-44&lt;/RefSource&gt;&lt;PMID Version="1"&gt;20619355&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Rev Genet. 2011 Jan;12(1):56-68&lt;/RefSource&gt;&lt;PMID Version="1"&gt;21164525&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Genet. 2011 Jun;43(6):513-8&lt;/RefSource&gt;&lt;PMID Version="1"&gt;21614091&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Rev Genet. 2011 Aug;12(8):529-41&lt;/RefSource&gt;&lt;PMID Version="1"&gt;21747404&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Rev Genet. 2011 Nov;12(11):745-55&lt;/RefSource&gt;&lt;PMID Version="1"&gt;21946919&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Am J Hum Genet. 2012 Jan 13;90(1):7-24&lt;/RefSource&gt;&lt;PMID Version="1"&gt;22243964&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;FEBS J. 2012 Mar;279(5):678-96&lt;/RefSource&gt;&lt;PMID Version="1"&gt;22221742&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nature. 2012 Sep 6;489(7414):57-74&lt;/RefSource&gt;&lt;PMID Version="1"&gt;22955616&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nature. 2012 Sep 6;489(7414):91-100&lt;/RefSource&gt;&lt;PMID Version="1"&gt;22955619&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Bioinformatics. 2012 Dec 1;28(23):3153-4&lt;/RefSource&gt;&lt;PMID Version="1"&gt;23044545&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nucleic Acids Res. 2013 Jan;41(Database issue):D991-5&lt;/RefSource&gt;&lt;PMID Version="1"&gt;23193258&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nucleic Acids Res. 2014 Jan;42(Database issue):D1001-6&lt;/RefSource&gt;&lt;PMID Version="1"&gt;24316577&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nature. 2014 Apr 24;508(7497):469-76&lt;/RefSource&gt;&lt;PMID Version="1"&gt;24759409&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Orphanet J Rare Dis. 2014;9:88&lt;/RefSource&gt;&lt;PMID Version="1"&gt;24948216&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nucleic Acids Res. 2015 Jan;43(Database issue):D799-804&lt;/RefSource&gt;&lt;PMID Version="1"&gt;25428361&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nucleic Acids Res. 2000 Jan 1;28(1):27-30&lt;/RefSource&gt;&lt;PMID Version="1"&gt;10592173&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Genet. 2000 May;25(1):25-9&lt;/RefSource&gt;&lt;PMID Version="1"&gt;10802651&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Neural Netw. 2000 May-Jun;13(4-5):411-30&lt;/RefSource&gt;&lt;PMID Version="1"&gt;10946390&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Annu Rev Biomed Eng. 2002;4:129-53&lt;/RefSource&gt;&lt;PMID Version="1"&gt;12117754&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Bioinformatics. 2003 Jan 22;19(2):185-93&lt;/RefSource&gt;&lt;PMID Version="1"&gt;12538238&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Biostatistics. 2003 Apr;4(2):249-64&lt;/RefSource&gt;&lt;PMID Version="1"&gt;12925520&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Genome Biol. 2003;4(11):R76&lt;/RefSource&gt;&lt;PMID Version="1"&gt;14611662&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;CommentsCorrections RefType="Cites"&gt;&lt;RefSource&gt;Nat Genet. 2005 Apr;37(4):413-7&lt;/RefSource&gt;&lt;PMID Version="1"&gt;15793588&lt;/PMID&gt;&lt;/CommentsCorrections&gt;&lt;/CommentsCorrectionsList&gt;&lt;MeshHeadingList&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D000465"&gt;Algorithms&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;G17.035&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.224.050&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D019295"&gt;Computational Biology&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000379"&gt;methods&lt;/QualifierName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000706"&gt;statistics &amp;amp; numerical data&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;H01.158.273.180&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.313.124&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y33&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y09&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D003920"&gt;Diabetes Mellitus&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000235"&gt;genetics&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;C18.452.394.750&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;C19.246&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y05.050&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y08.010&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D020869"&gt;Gene Expression Profiling&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.393.332&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D053263"&gt;Gene Regulatory Networks&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;G05.360.080.689.360&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="Y" UI="D020022"&gt;Genetic Predisposition to Disease&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="Y"&gt;C23.550.291.687.500&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="Y"&gt;G05.380.355&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D055106"&gt;Genome-Wide Association Study&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000706"&gt;statistics &amp;amp; numerical data&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.318.416.249&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.318.780.392&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.393.385.500&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.393.522.500&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.393.760.640.500&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;N06.850.520.445.392&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;N06.850.520.470.500&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y09&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D006801"&gt;Humans&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;B01.050.150.900.649.801.400.112.400.400&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D001523"&gt;Mental Disorders&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="N" UI="Q000235"&gt;genetics&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;F03&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y05.050&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;Y08.010&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D008957"&gt;Models, Genetic&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.599.395.397&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D020412"&gt;Multifactorial Inheritance&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;G05.420.590&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="Y" UI="D059467"&gt;Transcriptome&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="Y"&gt;G02.111.087.847.750&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="Y"&gt;G02.149.115.847.750&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="Y"&gt;G05.355.310.700.750&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="Y"&gt;G05.360.920&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D000069558"&gt;Unsupervised Machine Learning&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;G17.035.250.500.750&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.224.050.375.530.750&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;/MeshHeadingList&gt;&lt;OtherID Source="NLM"&gt;NIHMS742531&lt;/OtherID&gt;&lt;OtherID Source="NLM"&gt;PMC4722949&lt;/OtherID&gt;&lt;/item&gt;&lt;/static_data&gt;&lt;dynamic_data&gt;&lt;citation_related&gt;&lt;tc_list&gt;&lt;silo_tc coll_id="MEDLINE" local_count="0"&gt;&lt;/silo_tc&gt;&lt;/tc_list&gt;&lt;/citation_related&gt;&lt;cluster_related&gt;&lt;identifiers&gt;&lt;identifier type="issn" value="2335-6936"&gt;&lt;/identifier&gt;&lt;identifier type="pmid" value="MEDLINE:26776202"&gt;&lt;/identifier&gt;&lt;/identifiers&gt;&lt;/cluster_related&gt;&lt;/dynamic_data&gt;&lt;/REC&gt;
          &lt;REC r_id_disclaimer="ResearcherID data provided by Clarivate Analytics"&gt;&lt;UID&gt;MEDLINE:7584390&lt;/UID&gt;&lt;static_data&gt;&lt;summary&gt;&lt;EWUID&gt;&lt;WUID coll_id="MEDLINE"&gt;&lt;/WUID&gt;&lt;edition value="MEDLINE.MEDLINE"&gt;&lt;/edition&gt;&lt;/EWUID&gt;&lt;pub_info coverdate="1994" has_abstract="Y" medium="Print" model="Print" pubtype="Journal" pubyear="1994" sortdate="1994-01-01" vol="2"&gt;&lt;page begin="19" end="27" page_count="9"&gt;19-27&lt;/page&gt;&lt;/pub_info&gt;&lt;titles count="4"&gt;&lt;title type="item"&gt;Finding an average core structure: application to the globins.&lt;/title&gt;&lt;title type="source"&gt;Proceedings. International Conference on Intelligent Systems for Molecular Biology&lt;/title&gt;&lt;title type="abbrev_iso"&gt;Proc Int Conf Intell Syst Mol Biol&lt;/title&gt;&lt;title type="source_abbrev"&gt;Proc Int Conf Intell Syst Mol Biol&lt;/title&gt;&lt;/titles&gt;&lt;names count="2"&gt;&lt;name display="Y" role="author" seq_no="1"&gt;&lt;display_name&gt;Altman, R B&lt;/display_name&gt;&lt;full_name&gt;Altman, R B&lt;/full_name&gt;&lt;initials&gt;RB&lt;/initials&gt;&lt;/name&gt;&lt;name display="Y" role="author" seq_no="2"&gt;&lt;display_name&gt;Gerstein, M&lt;/display_name&gt;&lt;full_name&gt;Gerstein, M&lt;/full_name&gt;&lt;initials&gt;M&lt;/initials&gt;&lt;/name&gt;&lt;/names&gt;&lt;doctypes count="3"&gt;&lt;doctype&gt;Journal Article&lt;/doctype&gt;&lt;doctype&gt;Research Support, Non-U.S. Gov&amp;apos;t&lt;/doctype&gt;&lt;doctype&gt;Research Support, U.S. Gov&amp;apos;t, P.H.S.&lt;/doctype&gt;&lt;/doctypes&gt;&lt;/summary&gt;&lt;fullrecord_metadata&gt;&lt;languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/languages&gt;&lt;normalized_languages count="1"&gt;&lt;language type="primary"&gt;English&lt;/language&gt;&lt;/normalized_languages&gt;&lt;normalized_doctypes count="2"&gt;&lt;doctype&gt;Article&lt;/doctype&gt;&lt;doctype&gt;Other&lt;/doctype&gt;&lt;/normalized_doctypes&gt;&lt;addresses count="1"&gt;&lt;address_name&gt;&lt;address_spec addr_no="1"&gt;&lt;full_address&gt;Section on Medical Informatics, Stanford University, CA 94305, USA.&lt;/full_address&gt;&lt;/address_spec&gt;&lt;/address_name&gt;&lt;/addresses&gt;&lt;category_info&gt;&lt;headings count="1"&gt;&lt;heading&gt;Science &amp;amp; Technology&lt;/heading&gt;&lt;/headings&gt;&lt;subheadings count="3"&gt;&lt;subheading&gt;Physical Sciences&lt;/subheading&gt;&lt;subheading&gt;Life Sciences &amp;amp; Biomedicine&lt;/subheading&gt;&lt;subheading&gt;Technology&lt;/subheading&gt;&lt;/subheadings&gt;&lt;subjects count="10"&gt;&lt;subject ascatype="traditional" code="PQ"&gt;MATHEMATICS&lt;/subject&gt;&lt;subject ascatype="traditional"&gt;BIOCHEMISTRY MOLECULAR BIOLOGY&lt;/subject&gt;&lt;subject ascatype="traditional"&gt;COMPUTER SCIENCE INTERDISCIPLINARY APPLICATIONS&lt;/subject&gt;&lt;subject ascatype="traditional" code="MA"&gt;HEMATOLOGY&lt;/subject&gt;&lt;subject ascatype="traditional"&gt;GENETICS HEREDITY&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Mathematics&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Biochemistry &amp;amp; Molecular Biology&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Computer Science&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Hematology&lt;/subject&gt;&lt;subject ascatype="extended"&gt;Genetics &amp;amp; Heredity&lt;/subject&gt;&lt;/subjects&gt;&lt;/category_info&gt;&lt;fund_ack&gt;&lt;grants complete="Y" count="1"&gt;&lt;grant&gt;&lt;grant_agency&gt;NLM NIH HHS&lt;/grant_agency&gt;&lt;grant_ids count="1"&gt;&lt;grant_id&gt;LM-05305&lt;/grant_id&gt;&lt;/grant_ids&gt;&lt;country&gt;United States&lt;/country&gt;&lt;acronym&gt;LM&lt;/acronym&gt;&lt;/grant&gt;&lt;/grants&gt;&lt;/fund_ack&gt;&lt;abstracts count="1"&gt;&lt;abstract&gt;&lt;abstract_text&gt;&lt;p&gt;We present a procedure for automatically identifying from a set of aligned protein structures a subset of atoms with only a small amount of structural variation, i.e., a core. We apply this procedure to the globin family of proteins. Based purely on the results of the procedure, we show that the globin fold can be divided into two parts. The part with greater structural variation consists of the residues near the heme (the F helix and parts of the G and H helices), and the part with lesser structural variation (the core) forms a structural framework similar to that of the repressor protein (A, B, and E helices and remainder of the G and H helices). Such a division is consistent with many other structural and biochemical findings. In addition, we find further partitions within the core that may have biological significance. Finally, using the structural core of the globin family as a reference point, we have compared structural variation to sequence variation and shown that a core definition based on sequence conservation does not necessarily agree with one based on structural similarity.&lt;/p&gt;&lt;/abstract_text&gt;&lt;/abstract&gt;&lt;/abstracts&gt;&lt;/fullrecord_metadata&gt;&lt;item xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Owner="NLM" Status="MEDLINE" coll_id="MEDLINE" xsi:type="itemType_medline"&gt;&lt;MedlineJournalInfo&gt;&lt;Country&gt;United States&lt;/Country&gt;&lt;NlmUniqueID&gt;9509125&lt;/NlmUniqueID&gt;&lt;ISSNLinking&gt;1553-0833&lt;/ISSNLinking&gt;&lt;/MedlineJournalInfo&gt;&lt;DateCreated&gt;1995-12-14&lt;/DateCreated&gt;&lt;DateCompleted&gt;1995-12-14&lt;/DateCompleted&gt;&lt;DateRevised&gt;2016-10-21&lt;/DateRevised&gt;&lt;Affiliation&gt;Section on Medical Informatics, Stanford University, CA 94305, USA.&lt;/Affiliation&gt;&lt;ChemicalList&gt;&lt;Chemical&gt;&lt;RegistryNumber&gt;9004-22-2&lt;/RegistryNumber&gt;&lt;NameOfSubstance UI="D005914"&gt;Globins&lt;/NameOfSubstance&gt;&lt;/Chemical&gt;&lt;/ChemicalList&gt;&lt;CitationSubset&gt;IM&lt;/CitationSubset&gt;&lt;MeshHeadingList&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D000465"&gt;Algorithms&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;G17.035&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.224.050&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D000595"&gt;Amino Acid Sequence&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;G02.111.570.060&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.453.245.667.060&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D000818"&gt;Animals&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;B01.050&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D003198"&gt;Computer Simulation&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.224.160&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D005914"&gt;Globins&lt;/DescriptorName&gt;&lt;QualifierName MajorTopicYN="Y" UI="Q000737"&gt;chemistry&lt;/QualifierName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;D12.776.422.316&lt;/TreeCode&gt;&lt;TreeCode MajorTopicYN="Y"&gt;Y03&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D006801"&gt;Humans&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;B01.050.150.900.649.801.400.112.400.400&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D008969"&gt;Molecular Sequence Data&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;L01.453.245.667&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D016415"&gt;Sequence Alignment&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.393.751&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;MeshHeading&gt;&lt;DescriptorName MajorTopicYN="N" UI="D017421"&gt;Sequence Analysis&lt;/DescriptorName&gt;&lt;TreeCodeList&gt;&lt;TreeCode MajorTopicYN="N"&gt;E05.393.760&lt;/TreeCode&gt;&lt;/TreeCodeList&gt;&lt;/MeshHeading&gt;&lt;/MeshHeadingList&gt;&lt;/item&gt;&lt;/static_data&gt;&lt;dynamic_data&gt;&lt;citation_related&gt;&lt;tc_list&gt;&lt;silo_tc coll_id="MEDLINE" local_count="0"&gt;&lt;/silo_tc&gt;&lt;/tc_list&gt;&lt;/citation_related&gt;&lt;cluster_related&gt;&lt;identifiers&gt;&lt;identifier type="issn" value="1553-0833"&gt;&lt;/identifier&gt;&lt;identifier type="pmid" value="MEDLINE:7584390"&gt;&lt;/identifier&gt;&lt;/identifiers&gt;&lt;/cluster_related&gt;&lt;/dynamic_data&gt;&lt;/REC&gt;
  &lt;/records&gt;
